target/linux/ar71xx/generic/profiles/anonabox.mk

```diff
@@ -0,0 +1,18 @@
+#                                                                                                                                                                                     
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+define Profile/ANONABOX_PRO
+	NAME:=AnonaBox Pro board
+	PACKAGES:=kmod-usb-core kmod-usb-ohci kmod-usb-uhci \
```

Comment:
Less is more, this is most likely not going to be accepted as is. Have a look at the minimal config or similar ones.